### DISCUSSION TITLE
Solana

### DISCUSSION THREAD

**Discussion by GoldTaro** [Huniko-Team/community#202] [Q&A] [answered]
How does Solana's Tower BFT consensus mechanism work?

**Comment by Tvenus** [comment, marked as answer]
Tower BFT is Solana's implementation of Practical Byzantine Fault Tolerance (PBFT) optimized for PoH. It uses the PoH-generated timestamps to reduce communication overhead and speed up consensus, ensuring fast and secure transaction finality.
